recipes/alabaster/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.7.7" %}
+
+package:
+  name: alabaster
+  version: {{version}}
+
+source:
+  fn: alabaster-{{version}}.tar.gz
+  url: https://pypi.python.org/packages/source/a/alabaster/alabaster-{{version}}.tar.gz
```

Comment:
Could we please switch to pypi.io? It should save you time later.

Also, going from `{{version}}` to `{{ version }}` would be nice for readability.